src/neuroconv/tools/nwb_helpers/_configure_backend.py

```diff
@@ -24,12 +24,20 @@ def configure_backend(
         # TODO: update buffer shape in iterator, if present
 
         nwbfile_object = nwbfile_objects[object_id]
+        # Table columns
         if isinstance(nwbfile_object, Data):
             nwbfile_object.set_data_io(data_io_class=data_io_class, data_io_kwargs=data_io_kwargs)
-        elif isinstance(nwbfile_object, TimeSeries):
+        # TimeSeries data or timestamps
+        elif isinstance(nwbfile_object, TimeSeries) and not isinstance(
```

Comment:
I would create an explicit variable for this condition.
`is_dataset_linked = isinstance(nwbfile_object.fields.get(dataset_name)`

I think that explicitly naming the condition will capture intent of what we are trying to do (self-document) and make it easier to change in the future if we need to.

Reply:
Sounds good, added that for readability